src/frontend/web_application/src/components/ContactDetails/components/AddressForm/index.jsx

```diff
@@ -20,8 +21,6 @@ class AddressForm extends Component {
 
   constructor(props) {
     super(props);
-    this.handleSubmit = this.handleSubmit.bind(this);
-    this.handleInputChange = this.handleInputChange.bind(this);
     this.state = {
       contactDetail: {
```

Comment:
You might want to keep the `state` as flat as possible, remove all unnecessary namespaces

This component is good example because the only thing stored in the Component's state are "contact details", thus the `contactDetails` namespace is unnecessary and gets in the way of potential optimizations